Makefile

```diff
@@ -247,6 +247,7 @@ generate:
 mdatagen-test:
 	cd cmd/mdatagen && $(GOCMD) install .
 	cd cmd/mdatagen && $(GOCMD) generate ./...
+	cd cmd/mdatagen && $(GOCMD) test ./...
```

Comment:
Maybe better put it in the first place? Feel free to submit a separate PR to unblock the automated builds :)